src/app/lib/config/services/mundo.js

```diff
@@ -142,24 +142,24 @@ export const service = {
             },
           },
           accept: 'Sí, estoy de acuerdo',
-          reject: 'No, llévame a los ajustes',
+          reject: 'No, llévame a la configuración',
           rejectUrl:
             'https://www.bbc.co.uk/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
         },
       },
       media: {
         noJs:
-          'To play this content, please enable JavaScript, or try a different browser',
+          'Para ver este contenido, favor activar JavaScript, o intentar con otro navegador',
```

Comment:
Just a small tweak:
```suggestion
          'Para ver este contenido, por favor activar JavaScript, o intentar con otro navegador',
```
And I personally find this more accurate:
```suggestion
          'Para ver este contenido, por favor active JavaScript o intente con otro navegador',
```
```
```

Reply:
I'll run this past editorial. Thanks :)

Reply:
Editorial's view is that we should stick to the translation they provided, because their Latin American Spanish is closer to how their users speak, despite sounding weird to Spanish Spanish speakers.

Reply:
Fair enough, however `favor` is wrong on its own, it should be `por favor`.